manifests/cluster-manager/management/cluster-manager-manifestworkreplicaset-deployment.yaml

```diff
@@ -46,10 +46,15 @@ spec:
         args:
           - "/work"
           - "manager"
-          - "--work-driver=kube"
+          - "--work-driver={{ .WorkDriver }}"
+          {{ if eq .WorkDriver "kube" }}
           {{ if .HostedMode }}
           - "--kubeconfig=/var/run/secrets/hub/kubeconfig"
           {{ end }}
+          {{ else }}
+          - "--cloudevents-client-id=$(POD_NAME)"
```

Comment:
not sure if using podname is good choice. The pod name is changed after upgrade. Should the client id be already in the config file?

Reply:
the client id is not in config file, it's in cloudevents source options, see: https://github.com/open-cluster-management-io/sdk-go/blob/7dbdd1b5c63da7b832705501185138a5e5c530ba/pkg/cloudevents/generic/options/mqtt/sourceoptions.go#L21

per my understanding, it's acceptable if the client is changed, since it's used by MQTT broker to differentiate clients, the key is to make sure each publisher/subscriber has different client id.

Reply:
I am more concerned about if client id is part of authentication/authz, which will be impacted by the change of client id. cc @skeeey

Reply:
I remembered for some MQTT broker, like the AWS IOT, we need this client ID do authn/authz, but we can use a wildcard in the  authn/authz, I think you may need a fixed format ID here

Reply:
thanks, I updated this to `"--cloudevents-client-id=work-controller-$(POD_NAME)"` with fixed prefix `work-controller-`, so in the future the authn/zuthz can be donw with wildcard `work-controller-*`.